src/pudl/metadata/fields.py

```diff
@@ -998,6 +1034,14 @@
         "description": "Code identifying a dataset available within PUDL.",
         "constraints": {"enum": list(SOURCES)},
     },
+    "date_filed": {
+        "type": "datetime",
+        "description": "Date filing was submitted.",
+    },
+    "date_of_name_change": {
+        "type": "datetime",
+        "description": "Date of last name change of the company.",
+    },
```

Comment:
As above, the type, name, and description should all reflect the same resolution (date vs. datetime) and use `_date` or `_datetime_utc` (if UTC) as suffixes.

Reply:
Fixed date inconsistencies